crates/burn-import/pytorch-tests/tests/non_contiguous_indexes/mod.rs

```diff
@@ -0,0 +1,109 @@
+use burn::{
+    module::Module,
+    nn::{
+        conv::{Conv2d, Conv2dConfig},
+        PaddingConfig2d,
+    },
+    tensor::{activation::relu, backend::Backend, Tensor},
+};
+
+#[derive(Module, Debug)]
+pub struct Net<B: Backend> {
+    fc: Vec<Conv2d<B>>,
+}
+
+impl<B: Backend> Net<B> {
+    /// Create a new model from the given record.
+    pub fn new_with(record: NetRecord<B>) -> Self {
+        let conv2d_config = Conv2dConfig::new([2, 2], [3, 3]).with_padding(PaddingConfig2d::Same);
+        let mut fc = vec![];
+        for fc_record in record.fc.into_iter() {
+            fc.push(conv2d_config.init_with(fc_record));
+        }
+        Net { fc }
+    }
+
+    /// Forward pass of the model.
+    pub fn forward(&self, x: Tensor<B, 4>) -> Tensor<B, 4> {
+        let x = relu(self.fc[0].forward(x));
```

Comment:
If this serves as an example, we could change the forward to work with any number of elements in the vec sequence.

```rust
self.fc.iter().fold(x, |x_i, conv| relu(conv.forward(x_i)))
```